cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -149,3 +195,15 @@
     display: none;
   }
 });
+
+.u-hide-on-desktop {
+  .respond-to-min( @bp-med-min, {
+    display: none;
+  } );
+}
+
+.u-hide-on-mobile {
```

Comment:
[This already exists in cfpb-core](https://github.com/cfpb/design-system/blob/425fbf7e01fa072eaa39118c274ef24b351693b7/packages/cfpb-core/src/utilities.less#L57). Also, you could likely use the `.u-show-on-mobile` in core instead of `.u-hide-on-desktop` above.